app/Http/Controllers/ProfileController.php

```diff
@@ -16,8 +17,11 @@ class ProfileController extends Controller
      */
     public function edit(Request $request): View
     {
+        $user = $request->user();
+        $curriculo = Curriculo::where('user_id', $user->id)->first();
```

Comment:
Funciona bien, pero no queda muy claro a la hora de leerlo, con un findOrFail te curas en salud para los errores en caso de que el curriculo no este en la base de datos, por que con lo que tienes te da "Attempt to read property "id" on null" a la hora de abrir el edit de un usuario que se acabe de crear (Por poner un ejemplo)

Reply:
(Mas que funciona bien, funciona siempre que no le pongas un valor fuera del limite, no lo habia dejado muy claro)